.travis.yml

```diff
@@ -1,13 +1,19 @@
 language: rust
-sudo: false
```

Comment:
I believe `sudo: false` is the default, at least for the precise images. The documenation isn't consistent about what's required, of course. :/

I didn't see any difference in the worker description section or in dispatch speed with or without it.